WooCommerce/Resources/Images.xcassets/POS/pos_new_transaction_icon.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Added the icon in `POS` folder in the Images assets catalog

Reply:
nit: consider defining the custom image in UIImage+Woo with a test case in IconsTests to be consistent with other images